libraries/scopes/test/simplify_test.js

```diff
@@ -0,0 +1,33 @@
+const utils = require("../src");
+const testing = require("taskcluster-lib-testing");
+const assert = require("assert");
+
+suite(testing.suiteName(), function() {
+  test('single scope simplifies to itself', function() {
+    assert.deepStrictEqual(utils.simplifyScopeExpression("scope:1"), "scope:1");
+  });
+
+  test('AnyOf simplifies to itself', function() {
+    assert.deepStrictEqual(
+      utils.simplifyScopeExpression({AnyOf: ["scope:1", "scope:2"]}),
+      {AnyOf: ["scope:1", "scope:2"]});
+  });
+
+  test('AllOf simplifies to itself', function() {
+    assert.deepStrictEqual(
+      utils.simplifyScopeExpression({AllOf: ["scope:1", "scope:2"]}),
+      {AllOf: ["scope:1", "scope:2"]});
+  });
+
+  test('AnyOf list of scopes duplicates are removed', function() {
+    assert.notDeepEqual(
```

Comment:
"not"??

Reply:
Uh oh!, that should be `strictDeepEqual`